mkdocs.yml

```diff
@@ -128,6 +128,8 @@ plugins:
         'variants/protectli_vp2410/recovery.md': 'unified/protectli/recovery.md#vp2410'
         'variants/protectli_vp2420/recovery.md': 'unified/protectli/recovery.md#vp2420'
 
+        'osf-trivia-list/dasharo-entry-subscription.md': 'osf-trivia-list/dasharo-pro-package.md'
```

Comment:
We have changed more than one heading, this is obviously not enough

Reply:
Right, also the headings, not only files should be included here.